vita3k/renderer/include/renderer/vulkan/pipeline_cache.h

```diff
@@ -42,7 +43,7 @@ class PipelineCache {
 
     // does the GPU support vertex attributes with 3 components (like R16G16B16_UNORM), some (like AMD GPUs) don't
     // this is needed when creating the input state
-    bool support_rgb_vertex_attribute;
+    std::unordered_set<vk::Format> unsupported_rgb_vertex_attribute_formats{};
```

Comment:
A set should be faster than an unordered_set when it contains only a few values I believe.

Reply:
I can change it to set of bit flags in one byte, It will be fastest way, but code may be hard to understand and contains a lot of с style casts

Reply:
I was more thinking of using a std::set instead of a std::unordered_set. Using a set of bit flags would indeed be faster but I don't think this is necessary for the added complexity (also in this case using an std::bitset would remove the c casts).

Reply:
I fix set, Is there needed some more changes?